app/javascript/dashboard/store/modules/telegramDialogues.js

```
@@ -0,0 +1,406 @@
+      });
+      commit('SET_CHATS', chats);
+      commit('SET_CHATS_OFFSET', chats.length);
+      commit('SET_CHATS_HAS_MORE', chats.length >= CHATS_PAGE_SIZE);
+
+      const counts = {};
+      chats.forEach(c => {
+        if (c.unreadCount > 0) counts[c.id] = c.unreadCount;
+      });
+      commit('SET_UNREAD_COUNTS', counts);
+    } catch (error) {
+      console.error('[TelegramDialogues] Failed to fetch chats:', error);
```

ESLint

[error] Unexpected console statement.

(no-console)